### ISSUE TITLE
The netopeer2 debian package should depend on sysrepo-tools

### ISSUE THREAD

**Issue by cpascual** [CESNET/netopeer2#1668]
I am building the whole stack of netopeer2 and sysrepo debian packages using apkg.

[My created packages](https://github.com/cpascual/netopeer-deb) work fine except that the netopeer2 package fails to install unless the `sysrepo` group exists in the system.

The error can be reproduced in a fresh machine (e.g. by doing `docker run --rm -it debian:bookworm-slim bash`):

```console
$ wget https://cpascual.github.io/netopeer-deb/gpg.key -O- > /etc/apt/trusted.gpg.d/netopeer-deb.asc
$ echo 'deb [arch=amd64] https://cpascual.github.io/netopeer-deb/ bookworm main' > /etc/apt/sources.list.d/netopeer-deb.list
$ apt update && apt install -y netopeer2
...
The following additional packages will be installed:
  krb5-locales libbrotli1 libcurl4 libgssapi-krb5-2 libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 libldap-2.5-0 libldap-common libnetconf4 libnghttp2-14 librtmp1 libsasl2-2 libsasl2-modules
  libsasl2-modules-db libssh-4 libssh2-1 libsysrepo7 libyang3
...
Preparing to unpack .../20-netopeer2_2.2.31-1_amd64.deb ...
usermod: group 'sysrepo' does not exist
dpkg: error processing archive /tmp/apt-dpkg-install-RFdDKV/20-netopeer2_2.2.31-1_amd64.deb (--unpack):
 new netopeer2 package pre-installation script subprocess returned error exit status 6
Errors were encountered while processing:
 /tmp/apt-dpkg-install-RFdDKV/20-netopeer2_2.2.31-1_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

The `sysrepo` group is automatically created by the `sysrepo-tools` package. And since netopeer2 uses sysrepo anyway , I think it would make sense to add `sysrepo-tools` to "Depends" in [distro/pkg/deb/control](https://github.com/CESNET/netopeer2/blob/master/distro/pkg/deb/control).

If you agree, I can submit a PR.

**Comment by michalvasko** [comment]
I agree that `sysrepoctl` utility is used in the `postinst` script so yes, `sysrepo-tools` is definitely needed and should be in the dependencies. But the group `sysrepo` is correctly created by the `libsysrepo7` package, I have just checked.

**Comment by michalvasko** [comment]
I have just tested it on Ubuntu 22.04 and it worked as expected, after installing `libsysrepo7`, the group `sysrepo` was added. But I see that you are building for Debian 12 so perhaps the `postinst` script is not executing correctly there? You can check that and if you look into the created `libsysrepo7` DEB package, into `control.tar.zst`, you will find the `postinst` script there.

**Comment by cpascual** [comment]
Hi, you are 100% right. I just checked and in my packages the `sysrepo` group is also correctly created by `libsysrepo7`.

I was misguided by the traces where I saw the `usermod: group 'sysrepo' does not exist` error (which is probably harmless) and did not notice the real error cause:

`/usr/share/netopeer2/scripts/setup.sh: Unable to find sysrepoctl executable.`

Sorry for the confusion.

Still, we agree that sysrepo should be included in the depends of netopeer2.

Do you want me to create a PR for that?

**Comment by cpascual** [comment]
thanks for the super-quick resolution!
